### ISSUE TITLE
Ruffle Extension for Safari

### ISSUE THREAD

**Issue by navanchauhan** [ruffle-rs/ruffle#1634]
The current nightly builds are for Chrome or Firefox. I tested the Chrome extension on Safari by converting it with `safari-web-extension-converter` and it works perfectly! ( Well, not perfectly the only issue is a cosmetic error in the extension which could be easily fixed )

Conversion:

<img width="562" alt="image" src="https://user-images.githubusercontent.com/29234112/99871215-9ff12280-2bfe-11eb-8ad9-7d1f219f6c97.png">

Demo:

<img width="928" alt="image" src="https://user-images.githubusercontent.com/29234112/99871182-6ddfc080-2bfe-11eb-889e-57c78e51a52e.png">

( Stealth Hunter 2 is unplayable and just shows a black screen but this happens in the Ruffle Demo website too )

Uploaded the Ruffle Extension app; To test it on Safari you will need to enable `Allow Unsigned Extensions` from the `Develop` menu

[Ruffle.app.zip](https://github.com/ruffle-rs/ruffle/files/5577224/Ruffle.app.zip)

Should this be added in the Wiki as `Experimental`?

**Comment by parkerlreed** [comment]
The home page already lists it as Safari compatible. I assume this is known. 

The wiki is user editable so following the existing template you could add the steps right below Chrome and Firefox

**Comment by navanchauhan** [comment]
Yes I did see that on the home page. I will close this issue after editing the Wiki

**Comment by navanchauhan** [comment]
Even though I have edited the wiki, should a pre-built extension be provided?
